### ISSUE TITLE
"MIME type ('text/html') is not executable, and strict MIME type checking is enabled" in Chrome 64

### ISSUE THREAD

**Issue by sleavitt** [itsgoingd/clockwork#233]
The web view does not take into account the correct mime type for .js files in Clockwork\Web\Web (it only has a check for paths containing .css in them, but not .js), causing them to be served up with mime type 'text/html', and Chrome 64 refuses to execute them due to strict mime type checking.

This is easily fixed by amending the asset() method to read something like the following:

```php
        public function asset($path)
        {
                $path = $this->resolveAssetPath($path);

                if (! $path) return;

                switch (pathinfo($path, PATHINFO_EXTENSION)) {
                        case 'css':
                                $mime = 'text/css';
                                break;
                        case 'js':
                                $mime = 'application/javascript';
                                break;
                        default:
                                $mime = 'text/html';
                                break;
                }

                return [
                        'path' => $path,
                        'mime' => $mime
                ];
        }
```

I'm sure there are other ways that could be written, but the mime type should be based on the ending extension of the path IMO. I modified the method to read as above and it worked fine for me on PHP 7 and I am no longer experiencing errors in the Chrome console due to strict MIME type checking on .js files from Clockwork.

**Comment by itsgoingd** [comment]
Nice, do you want to PR this change?

**Comment by sleavitt** [comment]
I can. Is the approach I took acceptable for a PR, or would it need
reworking? I'll happily fork and submit a PR, just want to make sure I'm
not causing someone else to have to rework it first.

**Comment by itsgoingd** [comment]
Looks fine to me the way you wrote it.

**Comment by sleavitt** [comment]
PR #234 has been submitted.

**Comment by itsgoingd** [comment]
Released in Clockwork 2.1, thanks again!
